settingshelper.py

```diff
@@ -251,6 +251,7 @@ def _before_send(event, hint):
     ignore_logger('django.template')
     ignore_logger('pillowtop')
     ignore_logger('restore')
+    ignore_logger('kafka.conn')
```

Comment:
is this logger actually in use?

Reply:
https://sentry.io/organizations/dimagi/issues/1216197130/

Reply:
okay. i was searching for `kafka.conn` in HQ codebase but thats probably not correct. Is there a way to search in code base to see where it's being used?

Reply:
It is used the in the kafka library so you would have to search your installed python files as well

Reply:
thanks.